### PR TITLE
fix contact fix footer

### PR DESCRIPTION
**Pull request by TomNUSDS**
Minor tweaks so we can close some issues.

See: #220 #352

### CODE REVIEW

File: client/src/pages/contact.tsx [hunk 1]
```diff
@@ -12,18 +12,17 @@ interface ContactPageProps {
 
 const ContactPage = ({location}: ContactPageProps) => {
   const generalemail = 'screeningtool.feedback@usds.gov';
```
Comment (vim-usds) [inline client/src/pages/contact.tsx:14]: Can these two be camelCased?

File: client/src/pages/contact.tsx [hunk 2]
```diff
@@ -12,18 +12,17 @@ interface ContactPageProps {
...
 
       <J40MainGridContainer fullWidth={true}>
-        <AlertWrapper />
+        <AlertWrapper/>
       </J40MainGridContainer>
 
       <J40MainGridContainer>
-        <Grid row><Grid col>
-          <section className={'usa-prose'}>
+        <section className={'usa-prose'}>
```
Comment (vim-usds) [inline client/src/pages/contact.tsx:24]: On some of the pages I think I remember seeing the pattern of Container > row > col > section. Is there something special here that we have to break from that pattern?

**Review by vim-usds — CHANGES_REQUESTED** [review]
Looks great! Just a few minor questions.

File: client/src/pages/contact.tsx (hunk 2 above)
Comment (TomNUSDS) [reply client/src/pages/contact.tsx:24]: Yeah. I'm not sure the correct approach here.  HTML spec says: 

> The <section> HTML element represents a generic standalone section of a document, which doesn't have a more specific semantic element to represent it. Sections should always have a heading, with very few exceptions.

So, it's kind of like a higher level `<h#/><p/>` part of the document?

It sounds like it's a fallback for html5 if the content doesn't fit nicely into `<article>`, `<aside>`, `<main>`, `<nav>`, etc.

One challenge of using Trussworks react library is that I'm not aware of if/where it uses html5 tags.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section

Not sure how close we're sticking to html5 tags in the rest of our page output?

File: client/src/pages/contact.tsx (hunk 1 above)
Comment (TomNUSDS) [reply client/src/pages/contact.tsx:14]: Good catch, it should be. (again, seems like linter should be catching this?)

Comment (TomNUSDS) [reply client/src/pages/contact.tsx]: fixed

File: client/src/pages/contact.tsx (hunk 2 above)
Comment (TomNUSDS) [reply client/src/pages/contact.tsx]: Removed `<section>` since it's already in a `<main>`

**Comment by TomNUSDS** [comment]
Re-review when you have time?

**Review by vim-usds — APPROVED** [review]
LGTM

**Pull request merged**